tests/integration/ceph_smoke_test.go

```diff
@@ -92,6 +92,8 @@ func (s *SmokeSuite) SetupSuite() {
 		Mons:                      3,
 		SkipOSDCreation:           false,
 		EnableAdmissionController: true,
+		ConnectionsEncrypted:      true,
+		ConnectionsCompressed:     true,
```

Comment:
Is this going to work since we don't run Quincy? Or perhaps we should only activate this against the daily nightly master job?

Reply:
If it's not quincy, I was testing with just logging a message instead of failing. I need to change it to an error and disable this setting in the test for now.

Reply:
I settled on logging an error if it's not quincy, so we can just keep this enabled in the tests and the daily nightly job will actually enable it.

Reply:
Still TODO?